src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/PopulationOperations.java

```diff
@@ -0,0 +1,128 @@
+    }
+
+    /**
+     * Returns a single population.
+     *
+     * @param id ID of the population
+     *
+     * @return The population
+     */
+    public Optional<Population> getPopulation(int id) {
+        // Join is more complicated and the performance gain would be negligible considering the the
+        // expected moderate usage
+        return create.fetchOptional(POPULATION, Tables.POPULATION.ID_POPULATION.eq(id))
+                .map(populationRecord -> {
+                    List<PopulationAnswerOptionRecord> answers = create.selectFrom(POPULATION_ANSWER_OPTION)
+                            .where(POPULATION_ANSWER_OPTION.POPULATION.eq(populationRecord.getIdPopulation()))
+                            .fetch();
+
+                    return PopulationTransform.toProto(populationRecord, answers);
+                });
+    }
+
+    /**
+     * Creates a new population.
+     *
+     * @param toStore Population to save
+     *
+     * @return Population with ID assigned
+     *
+     * @throws IllegalArgumentException if the name or content is not set
+     */
+    public Population insertPopulation(Population toStore) throws IllegalArgumentException {
+        assertHasField(toStore,
+                Population.QUESTION_FIELD_NUMBER,
+                Population.ANSWERS_FIELD_NUMBER);
+
+        PopulationRecord population = PopulationTransform.mergeRecord(create.newRecord(POPULATION), toStore);
+        population.store();
+
+        int[] primaryKey = toStore.getAnswersList().stream()
+                .map(s -> new PopulationAnswerOptionRecord(null, population.getIdPopulation(), s))
+                .collect(Collectors.collectingAndThen(Collectors.toList(), create::batchInsert))
+                .execute();
+
+        Result<PopulationAnswerOptionRecord> answerRecords = create
+                .selectFrom(POPULATION_ANSWER_OPTION)
+                .where(POPULATION_ANSWER_OPTION.ID_POPULATION_ANSWER_OPTION.in(Arrays.asList(primaryKey)))
+                .fetch();
+
+        return PopulationTransform.toProto(population, answerRecords);
+    }
+
+    /**
+     * Deletes a population.
+     *
+     * @param id ID of the population
+     *
+     * @return {@code true} if deleted, {@code false} otherwise
+     * @throws IllegalArgumentException if the population is still in use
+     */
+    public boolean deletePopulation(int id) throws IllegalArgumentException{
+        boolean isUsed = create.fetchExists(
+                DSL.select()
+                        .from(EXPERIMENTSPOPULATION)
+                        .join(POPULATION_ANSWER_OPTION).onKey()
+                        .where(POPULATION_ANSWER_OPTION.POPULATION.eq(id))
+        );
+
+        if (isUsed) {
+            throw new IllegalArgumentException(String.format("Population %d is still in used", id));
+        }
+
+        return create.deleteFrom(POPULATION)
+                .where(POPULATION.ID_POPULATION.eq(id))
+                .execute() == 1;
+    }
```

Comment:
@LeanderK Are the corresponding answer options automatically deleted by constraints?